CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # tulip changelog
 
 
+## 1.3.1
```

Comment:
The relevant comment about version numbers from #158 is <https://github.com/tulip-control/tulip-control/issues/158#issuecomment-243085416>, which recommends version 1.4.0 upon removal.